client/components/modal/index.css

```diff
@@ -16,7 +16,7 @@
 }
 .message {
   max-height: 70vh;
-  overflow: auto;
+  overflow: hidden;
```

Comment:
This change doesn't fix the issue. Please see #457 on smaller and wide screen sizes and you will see why scrollbar is shown.